packages/playground/wordpress-builds/build/build.js

```diff
@@ -101,7 +137,11 @@ try {
 	versions = {};
 }
 
-if (!args.force && versionInfo.slug !== 'nightly' && versions[versionInfo.slug] === versionInfo.version) {
+if (
+	!args.force &&
+	versionInfo.slug !== 'nightly' &&
```

Comment:
Unrelated to this issue. If we want to prevent ni0ghtly rebuilds from rerunning, we could store the nightly version as `nighty-YEAR-MONTH-DATE` instead `nightly`. This would prevent a rebuild if a nightly version was already built.

Reply:
I wonder if we could have the logged nightly version track the latest commit on WP trunk or something similar and only rebuild if that has changed.

Reply:
Where would we pull this data from? In GitHub WordPress just [keeps pushing commits through the day](https://github.com/WordPress/WordPress/commits/master/). 

Are there any disadvantages to using a date?

Reply:
Thanks, good catch! Fixed in 3a98021e96a67f2807b4071ca26cad4e2cd8e1cc

Reply:
Ah, I should've voiced my thoughts about the date suggestion. Sorry about that. It looks like we run nightly once a day [at the 9th hour](https://github.com/WordPress/wordpress-playground/blob/a91cbce7a33d5ef4e67927bbf875da4aebcbe786/.github/workflows/refresh-wordpress-nightly.yml#L7), so avoiding rebuilds within the same day didn't seem helpful.

Reply:
Makes sense! It sounds like the current nightly implementation is ok as is.